arm/Microsoft.Synapse/privateLinkHubs/.parameters/parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "value": "synplh0001"
+    },
+    "location": {
+      "value": "westeurope"
```

Comment:
Please also add tests for at least RBAC to the parameter file. You can copy the parameters from for example StorageAccount